### DISCUSSION TITLE
Brand new installation, doing intial import of external library getting hundreds of machine learning errors

### DISCUSSION THREAD

**Discussion by kingfenix** [immich-app/immich#10932] [Q&A]
### The bug

Importing external library for first time after installing using SpaceInvaderOne video and postgres/immich container along with immich container. import seems to be continuing but am getting hundreds or thousands of machine learning errors as it goes, is this normal?

### The OS that Immich Server is running on

Unraid Docker

### Version of Immich Server

latest

### Version of Immich Mobile App

latest

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
n/a
```

### Your .env content

```Shell
n/a
```

### Reproduction steps

```bash
begin first import on large 25 year photo library and then check logs in unraid
```

### Relevant log output

```shell
[Nest] 624  - 07/06/2024, 2:58:59 PM     LOG [Microservices:MediaService] Successfully generated JPEG image preview for asset 3e2e1336-7b0c-4637-a207-51f5da512376
[Nest] 624  - 07/06/2024, 2:58:59 PM   ERROR [Microservices:JobService] Unable to run job handler (smartSearch/smart-search): Error: Machine learning request '{"clip":{"visual":{"modelName":"ViT-B-32__openai"}}}' failed with status 500: Internal Server Error
[Nest] 624  - 07/06/2024, 2:58:59 PM   ERROR [Microservices:JobService] Error: Machine learning request '{"clip":{"visual":{"modelName":"ViT-B-32__openai"}}}' failed with status 500: Internal Server Error
    at MachineLearningRepository.predict (/app/immich/server/dist/repositories/machine-learning.repository.js:22:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async MachineLearningRepository.encodeImage (/app/immich/server/dist/repositories/machine-learning.repository.js:42:26)
    at async SmartInfoService.handleEncodeClip (/app/immich/server/dist/services/smart-info.service.js:91:27)
    at async /app/immich/server/dist/services/job.service.js:148:36
    at async Worker.processJob (/app/immich/server/node_modules/bullmq/dist/cjs/classes/worker.js:394:28)
    at async Worker.retryIfFailed (/app/immich/server/node_modules/bullmq/dist/cjs/classes/worker.js:581:24)
[Nest] 624  - 07/06/2024, 2:58:59 PM   ERROR [Microservices:JobService] Object:
```

**Comment by bo0tzz** [comment]
What do the logs of the ML container say? Do please post your compose file and .env as well, they are in fact applicable.

**Comment by kingfenix** [comment]
I dont have a compose or env file i couldnt get immich method to work that way, i had specified i followed spaceinvaderone’s video to install. I was hoping you might recognize that method i am using two docker containers one that he made which is a premade postgress and the immich with built in redis
